README.md

```diff
@@ -24,26 +26,25 @@ Basedmypy can be installed using pip from PyPI or from this GitHub repo:
     python -m pip install -U basedmypy
 
 ### Running
-Basedmypy currently overrides the `mypy` installation:
+Basedmypy is a direct alternative to the `mypy` installation:
```

Comment:
this seems misleading. it does override the mypy installation, which i think is a limitation that's very important to mention

Reply:
hmm

Reply:
Hows that look?

Reply:
that's ok i guess but i think it was fine how it was originally, less verbose and more clear imo

Reply:
I feel like "overrides" could be misleading, indicating that in needs to override mypy to be installed or something.

Reply:
"in place of" makes more sense

Reply:
> I feel like "overrides" could be misleading, indicating that in needs to override mypy to be installed or something.

but it does, doesn't it? though i guess "in place of" means pretty much the same thing so either is fine ig